in_toto/user_settings.py

```diff
@@ -0,0 +1,132 @@
+"""
+<Program Name>
+  user_settings.py
+
+<Author>
+  Lukas Puehringer <lukas.puehringer@nyu.edu>
+
+<Started>
+  Oct 25, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Provides methods to read environment variables and rcfiles and to write
+  them to the in_toto.settings module (override them).
+
+  Notes:
+  - Variable values are converted to lists if they contain colons
```

Comment:
In a comment, you should explain why this convention is used.   It is arbitrarily chosen, is it to ease parsing, to avoid parsing issues, or because multiple values are set this way with environment variables?